packages/botonic-core/src/hubtype-service.ts

```diff
@@ -1,24 +1,42 @@
-import axios from 'axios'
-import Pusher from 'pusher-js'
-
+import axios, { AxiosPromise } from 'axios'
+import Pusher, { Channel } from 'pusher-js'
+
+import {
+  BotonicHeaders,
+  HubtypeServiceArgs,
+  InitArgs,
+  ServerConfig,
+  SessionUser,
+} from './types'
 import { getWebpackEnvVar } from './utils'
 
 const _WEBCHAT_PUSHER_KEY_ = getWebpackEnvVar(
-  // eslint-disable-next-line no-undef
+  //@ts-ignore WEBPACK_VAR
   typeof WEBCHAT_PUSHER_KEY !== 'undefined' && WEBCHAT_PUSHER_KEY,
```

Comment:
I don't fully remember if we can sort this out with a declaration of this variable if we know it will be there.
```
declare const WEBCHAT_PUSHER_KEY: string
```

Reply:
sure! this is what declare stands for in fact, adeed :)